Columns available: Index(['Open', 'High', 'Low', 'Close', 'Adj Close', 'Volume'], dtype='object')
Epoch 1/50
WARNING:absl:At this time, the v2.11+ optimizer `tf.keras.optimizers.Adam` runs slowly on M1/M2 Macs, please use the legacy Keras optimizer instead, located at `tf.keras.optimizers.legacy.Adam`.
203/203 [==============================] - 5s 10ms/step - loss: 2.6776e-04 - val_loss: 9.5263e-05
Epoch 2/50
203/203 [==============================] - 2s 8ms/step - loss: 3.4107e-05 - val_loss: 4.6360e-06
Epoch 3/50
203/203 [==============================] - 2s 7ms/step - loss: 2.6914e-05 - val_loss: 3.1097e-05
Epoch 4/50
203/203 [==============================] - 1s 7ms/step - loss: 2.2003e-05 - val_loss: 4.1629e-05
Epoch 5/50
203/203 [==============================] - 2s 8ms/step - loss: 2.0656e-05 - val_loss: 1.5445e-05
Epoch 6/50
203/203 [==============================] - 2s 8ms/step - loss: 1.7312e-05 - val_loss: 7.0796e-06
Epoch 7/50
203/203 [==============================] - 2s 8ms/step - loss: 1.9126e-05 - val_loss: 3.1884e-05
Training completed with early stopping to prevent overfitting.
1/1 [==============================] - 1s 515ms/step
All data for ZBRA saved successfully to Excel.
Epoch 1/50
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/fromnumeric.py:3464: RuntimeWarning: Mean of empty slice.
  return _methods._mean(a, axis=axis, dtype=dtype,
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:192: RuntimeWarning: invalid value encountered in divide
  ret = ret.dtype.type(ret / rcount)
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:269: RuntimeWarning: Degrees of freedom <= 0 for slice
  ret = _var(a, axis=axis, dtype=dtype, out=out, ddof=ddof,
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:226: RuntimeWarning: invalid value encountered in divide
  arrmean = um.true_divide(arrmean, div, out=arrmean,
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:261: RuntimeWarning: invalid value encountered in divide
  ret = ret.dtype.type(ret / rcount)
WARNING:absl:At this time, the v2.11+ optimizer `tf.keras.optimizers.Adam` runs slowly on M1/M2 Macs, please use the legacy Keras optimizer instead, located at `tf.keras.optimizers.legacy.Adam`.
203/203 [==============================] - 5s 11ms/step - loss: 2.1745e-04 - val_loss: 2.0507e-05
Epoch 2/50
203/203 [==============================] - 2s 8ms/step - loss: 3.4096e-05 - val_loss: 1.9281e-05
Epoch 3/50
203/203 [==============================] - 2s 8ms/step - loss: 2.6731e-05 - val_loss: 2.1828e-06
Epoch 4/50
203/203 [==============================] - 2s 8ms/step - loss: 2.3820e-05 - val_loss: 7.2975e-05
Epoch 5/50
203/203 [==============================] - 1s 7ms/step - loss: 2.3067e-05 - val_loss: 5.2023e-05
Epoch 6/50
203/203 [==============================] - 1s 7ms/step - loss: 2.3415e-05 - val_loss: 9.1333e-06
Epoch 7/50
203/203 [==============================] - 2s 7ms/step - loss: 1.6046e-05 - val_loss: 1.2808e-04
Epoch 8/50
203/203 [==============================] - 1s 7ms/step - loss: 1.8864e-05 - val_loss: 4.6962e-05
Training completed with early stopping to prevent overfitting.
1/1 [==============================] - 1s 517ms/step
All data for ZBRA saved successfully to Excel.
Epoch 1/50
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/fromnumeric.py:3464: RuntimeWarning: Mean of empty slice.
  return _methods._mean(a, axis=axis, dtype=dtype,
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:192: RuntimeWarning: invalid value encountered in divide
  ret = ret.dtype.type(ret / rcount)
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:269: RuntimeWarning: Degrees of freedom <= 0 for slice
  ret = _var(a, axis=axis, dtype=dtype, out=out, ddof=ddof,
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:226: RuntimeWarning: invalid value encountered in divide
  arrmean = um.true_divide(arrmean, div, out=arrmean,
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:261: RuntimeWarning: invalid value encountered in divide
  ret = ret.dtype.type(ret / rcount)
WARNING:absl:At this time, the v2.11+ optimizer `tf.keras.optimizers.Adam` runs slowly on M1/M2 Macs, please use the legacy Keras optimizer instead, located at `tf.keras.optimizers.legacy.Adam`.
203/203 [==============================] - 4s 10ms/step - loss: 2.2066e-04 - val_loss: 8.1926e-05
Epoch 2/50
203/203 [==============================] - 1s 7ms/step - loss: 3.4382e-05 - val_loss: 1.4364e-05
Epoch 3/50
203/203 [==============================] - 2s 8ms/step - loss: 2.7931e-05 - val_loss: 5.8615e-05
Epoch 4/50
203/203 [==============================] - 1s 7ms/step - loss: 1.9525e-05 - val_loss: 2.4983e-05
Epoch 5/50
203/203 [==============================] - 1s 7ms/step - loss: 1.8348e-05 - val_loss: 1.1318e-04
Epoch 6/50
203/203 [==============================] - 1s 7ms/step - loss: 1.8941e-05 - val_loss: 1.2167e-04
Epoch 7/50
203/203 [==============================] - 2s 8ms/step - loss: 1.8440e-05 - val_loss: 1.3222e-04
Training completed with early stopping to prevent overfitting.
1/1 [==============================] - 1s 533ms/step
All data for ZBRA saved successfully to Excel.
Epoch 1/50
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/fromnumeric.py:3464: RuntimeWarning: Mean of empty slice.
  return _methods._mean(a, axis=axis, dtype=dtype,
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:192: RuntimeWarning: invalid value encountered in divide
  ret = ret.dtype.type(ret / rcount)
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:269: RuntimeWarning: Degrees of freedom <= 0 for slice
  ret = _var(a, axis=axis, dtype=dtype, out=out, ddof=ddof,
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:226: RuntimeWarning: invalid value encountered in divide
  arrmean = um.true_divide(arrmean, div, out=arrmean,
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:261: RuntimeWarning: invalid value encountered in divide
  ret = ret.dtype.type(ret / rcount)
WARNING:absl:At this time, the v2.11+ optimizer `tf.keras.optimizers.Adam` runs slowly on M1/M2 Macs, please use the legacy Keras optimizer instead, located at `tf.keras.optimizers.legacy.Adam`.
203/203 [==============================] - 4s 10ms/step - loss: 2.6022e-04 - val_loss: 1.1099e-07
Epoch 2/50
203/203 [==============================] - 2s 8ms/step - loss: 3.6830e-05 - val_loss: 1.3240e-08
Epoch 3/50
203/203 [==============================] - 2s 8ms/step - loss: 2.6816e-05 - val_loss: 2.9904e-07
Epoch 4/50
203/203 [==============================] - 1s 7ms/step - loss: 2.5941e-05 - val_loss: 4.4882e-05
Epoch 5/50
203/203 [==============================] - 1s 7ms/step - loss: 1.8811e-05 - val_loss: 2.4799e-05
Epoch 6/50
203/203 [==============================] - 1s 7ms/step - loss: 1.8636e-05 - val_loss: 5.0060e-08
Epoch 7/50
203/203 [==============================] - 2s 8ms/step - loss: 2.0006e-05 - val_loss: 4.7525e-07
Training completed with early stopping to prevent overfitting.
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/fromnumeric.py:3464: RuntimeWarning: Mean of empty slice.
  return _methods._mean(a, axis=axis, dtype=dtype,
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:192: RuntimeWarning: invalid value encountered in divide
  ret = ret.dtype.type(ret / rcount)
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:269: RuntimeWarning: Degrees of freedom <= 0 for slice
  ret = _var(a, axis=axis, dtype=dtype, out=out, ddof=ddof,
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:226: RuntimeWarning: invalid value encountered in divide
  arrmean = um.true_divide(arrmean, div, out=arrmean,
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:261: RuntimeWarning: invalid value encountered in divide
  ret = ret.dtype.type(ret / rcount)
1/1 [==============================] - 1s 558ms/step
All data for ZBRA saved successfully to Excel.
Epoch 1/50
WARNING:absl:At this time, the v2.11+ optimizer `tf.keras.optimizers.Adam` runs slowly on M1/M2 Macs, please use the legacy Keras optimizer instead, located at `tf.keras.optimizers.legacy.Adam`.
203/203 [==============================] - 5s 10ms/step - loss: 2.3705e-04 - val_loss: 1.9836e-05
Epoch 2/50
203/203 [==============================] - 2s 7ms/step - loss: 3.5973e-05 - val_loss: 8.9456e-06
Epoch 3/50
203/203 [==============================] - 2s 8ms/step - loss: 2.8673e-05 - val_loss: 9.2211e-06
Epoch 4/50
203/203 [==============================] - 2s 10ms/step - loss: 2.3162e-05 - val_loss: 2.1596e-06
Epoch 5/50
203/203 [==============================] - 2s 8ms/step - loss: 2.2032e-05 - val_loss: 2.6933e-06
Epoch 6/50
203/203 [==============================] - 2s 9ms/step - loss: 1.8194e-05 - val_loss: 1.0640e-06
Epoch 7/50
203/203 [==============================] - 2s 8ms/step - loss: 1.8596e-05 - val_loss: 1.3081e-05
Epoch 8/50
203/203 [==============================] - 1s 7ms/step - loss: 1.6659e-05 - val_loss: 6.9090e-06
Epoch 9/50
203/203 [==============================] - 1s 7ms/step - loss: 1.6525e-05 - val_loss: 6.0222e-05
Epoch 10/50
203/203 [==============================] - 1s 7ms/step - loss: 2.2164e-05 - val_loss: 1.3650e-05
Epoch 11/50
203/203 [==============================] - 1s 7ms/step - loss: 2.0331e-05 - val_loss: 3.0039e-05
Training completed with early stopping to prevent overfitting.
1/1 [==============================] - 1s 516ms/step
All data for ZBRA saved successfully to Excel.
Epoch 1/50
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/fromnumeric.py:3464: RuntimeWarning: Mean of empty slice.
  return _methods._mean(a, axis=axis, dtype=dtype,
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:192: RuntimeWarning: invalid value encountered in divide
  ret = ret.dtype.type(ret / rcount)
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:269: RuntimeWarning: Degrees of freedom <= 0 for slice
  ret = _var(a, axis=axis, dtype=dtype, out=out, ddof=ddof,
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:226: RuntimeWarning: invalid value encountered in divide
  arrmean = um.true_divide(arrmean, div, out=arrmean,
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:261: RuntimeWarning: invalid value encountered in divide
  ret = ret.dtype.type(ret / rcount)
WARNING:absl:At this time, the v2.11+ optimizer `tf.keras.optimizers.Adam` runs slowly on M1/M2 Macs, please use the legacy Keras optimizer instead, located at `tf.keras.optimizers.legacy.Adam`.
203/203 [==============================] - 5s 10ms/step - loss: 2.0253e-04 - val_loss: 1.2596e-05
Epoch 2/50
203/203 [==============================] - 1s 7ms/step - loss: 3.2958e-05 - val_loss: 1.7075e-06
Epoch 3/50
203/203 [==============================] - 1s 7ms/step - loss: 2.5965e-05 - val_loss: 2.4151e-05
Epoch 4/50
203/203 [==============================] - 2s 8ms/step - loss: 2.3480e-05 - val_loss: 6.7764e-06
Epoch 5/50
116/203 [================>.............] - ETA: 0s - loss: 1.7988e-05
169/203 [=======================>......] - ETA: 0s - loss: 1.9735e-05e-05 - val_loss: 6.7764e-06
 15/203 [=>............................] - ETA: 1s - loss: 1.1133e-05e-05 - val_loss: 1.0379e-05
203/203 [==============================] - 2s 8ms/step - loss: 1.8800e-05 - val_loss: 4.7476e-05
  5/203 [..............................] - ETA: 7s - loss: 2.4291e-05 -05 - val_loss: 4.7476e-05
177/203 [=========================>....] - ETA: 0s - loss: 1.7063e-05 -05 - val_loss: 4.7476e-05
203/203 [==============================] - 2s 12ms/step - loss: 1.6934e-05 - val_loss: 8.7006e-06
203/203 [==============================] - 2s 12ms/step - loss: 1.6934e-05 - val_loss: 8.7006e-06
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/fromnumeric.py:3464: RuntimeWarning: Mean of empty slice.
  return _methods._mean(a, axis=axis, dtype=dtype,
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:192: RuntimeWarning: invalid value encountered in divide
  ret = ret.dtype.type(ret / rcount)
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:269: RuntimeWarning: Degrees of freedom <= 0 for slice
  ret = _var(a, axis=axis, dtype=dtype, out=out, ddof=ddof,
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:226: RuntimeWarning: invalid value encountered in divide
  arrmean = um.true_divide(arrmean, div, out=arrmean,
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:261: RuntimeWarning: invalid value encountered in divide
Epoch 1/50=============================] - 2s 12ms/step - loss: 1.6934e-05 - val_loss: 8.7006e-06
Epoch 1/50=============================] - 2s 12ms/step - loss: 1.6934e-05 - val_loss: 8.7006e-06
WARNING:absl:At this time, the v2.11+ optimizer `tf.keras.optimizers.Adam` runs slowly on M1/M2 Macs, please use the legacy Keras optimizer instead, located at `tf.keras.optimizers.legacy.Adam`.
Epoch 1/50=============================] - 2s 12ms/step - loss: 1.6934e-05 - val_loss: 8.7006e-06
Epoch 3/50=============================] - 2s 12ms/step - loss: 1.6934e-05 - val_loss: 8.7006e-06
Epoch 5/50=============================] - 2s 12ms/step - loss: 1.6934e-05 - val_loss: 8.7006e-06
Epoch 5/50=============================] - 2s 12ms/step - loss: 1.6934e-05 - val_loss: 8.7006e-06
Epoch 7/50=============================] - 2s 12ms/step - loss: 1.6934e-05 - val_loss: 8.7006e-06
Training completed with early stopping to prevent overfitting.: 1.6934e-05 - val_loss: 8.7006e-06
Training completed with early stopping to prevent overfitting.: 1.6934e-05 - val_loss: 8.7006e-06
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/fromnumeric.py:3464: RuntimeWarning: Mean of empty slice.
  return _methods._mean(a, axis=axis, dtype=dtype,
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:192: RuntimeWarning: invalid value encountered in divide
  ret = ret.dtype.type(ret / rcount)
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:269: RuntimeWarning: Degrees of freedom <= 0 for slice
  ret = _var(a, axis=axis, dtype=dtype, out=out, ddof=ddof,
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:226: RuntimeWarning: invalid value encountered in divide
  arrmean = um.true_divide(arrmean, div, out=arrmean,
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:261: RuntimeWarning: invalid value encountered in divide
  ret = ret.dtype.type(ret / rcount)
Training completed with early stopping to prevent overfitting.: 1.6934e-05 - val_loss: 8.7006e-06
 66/203 [========>.....................] - ETA: 0s - loss: 5.0231e-044e-05 - val_loss: 8.7006e-06
203/203 [==============================] - 7s 9ms/step - loss: 1.9881e-04 - val_loss: 2.5559e-056
182/203 [=========================>....] - ETA: 0s - loss: 2.7380e-05e-04 - val_loss: 2.5559e-056
 72/203 [=========>....................] - ETA: 0s - loss: 2.8071e-05e-05 - val_loss: 1.3099e-046
203/203 [==============================] - 1s 7ms/step - loss: 2.3740e-05 - val_loss: 7.4349e-056
203/203 [==============================] - 1s 7ms/step - loss: 2.0788e-05 - val_loss: 1.7206e-056
137/203 [===================>..........] - ETA: 0s - loss: 1.7830e-05e-05 - val_loss: 1.7206e-056
 33/203 [===>..........................] - ETA: 1s - loss: 1.4436e-05e-05 - val_loss: 7.3403e-076
203/203 [==============================] - 1s 7ms/step - loss: 1.8691e-05 - val_loss: 9.9450e-056
203/203 [==============================] - 1s 7ms/step - loss: 1.7286e-05 - val_loss: 1.9904e-056
1/1 [==============================] - 1s 515ms/stepep - loss: 1.7286e-05 - val_loss: 1.9904e-056
1/1 [==============================] - 1s 515ms/stepep - loss: 1.7286e-05 - val_loss: 1.9904e-056
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/fromnumeric.py:3464: RuntimeWarning: Mean of empty slice.
  return _methods._mean(a, axis=axis, dtype=dtype,
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:192: RuntimeWarning: invalid value encountered in divide
  ret = ret.dtype.type(ret / rcount)
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:269: RuntimeWarning: Degrees of freedom <= 0 for slice
  ret = _var(a, axis=axis, dtype=dtype, out=out, ddof=ddof,
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:226: RuntimeWarning: invalid value encountered in divide
  arrmean = um.true_divide(arrmean, div, out=arrmean,
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:261: RuntimeWarning: invalid value encountered in divide
  ret = ret.dtype.type(ret / rcount)
1/1 [==============================] - 1s 515ms/stepep - loss: 1.7286e-05 - val_loss: 1.9904e-056
WARNING:absl:At this time, the v2.11+ optimizer `tf.keras.optimizers.Adam` runs slowly on M1/M2 Macs, please use the legacy Keras optimizer instead, located at `tf.keras.optimizers.legacy.Adam`.
Epoch 2/50=========================] - 1s 515ms/stepep - loss: 1.7286e-05 - val_loss: 1.9904e-056
Epoch 4/50=========================] - 1s 515ms/stepep - loss: 1.7286e-05 - val_loss: 1.9904e-056
Epoch 4/50=========================] - 1s 515ms/stepep - loss: 1.7286e-05 - val_loss: 1.9904e-056
Epoch 6/50=========================] - 1s 515ms/stepep - loss: 1.7286e-05 - val_loss: 1.9904e-056
Epoch 8/50=========================] - 1s 515ms/stepep - loss: 1.7286e-05 - val_loss: 1.9904e-056
Epoch 8/50=========================] - 1s 515ms/stepep - loss: 1.7286e-05 - val_loss: 1.9904e-056
Epoch 10/50========================] - 1s 515ms/stepep - loss: 1.7286e-05 - val_loss: 1.9904e-056
Epoch 12/50========================] - 1s 515ms/stepep - loss: 1.7286e-05 - val_loss: 1.9904e-056
Epoch 14/50========================] - 1s 515ms/stepep - loss: 1.7286e-05 - val_loss: 1.9904e-056
Epoch 14/50========================] - 1s 515ms/stepep - loss: 1.7286e-05 - val_loss: 1.9904e-056
Epoch 16/50========================] - 1s 515ms/stepep - loss: 1.7286e-05 - val_loss: 1.9904e-056
All data for ZBRA saved successfully to Excel.s/stepep - loss: 1.7286e-05 - val_loss: 1.9904e-056
All data for ZBRA saved successfully to Excel.s/stepep - loss: 1.7286e-05 - val_loss: 1.9904e-056
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/fromnumeric.py:3464: RuntimeWarning: Mean of empty slice.
  return _methods._mean(a, axis=axis, dtype=dtype,
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:192: RuntimeWarning: invalid value encountered in divide
  ret = ret.dtype.type(ret / rcount)
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:269: RuntimeWarning: Degrees of freedom <= 0 for slice
  ret = _var(a, axis=axis, dtype=dtype, out=out, ddof=ddof,
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:226: RuntimeWarning: invalid value encountered in divide
  arrmean = um.true_divide(arrmean, div, out=arrmean,
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:261: RuntimeWarning: invalid value encountered in divide
  ret = ret.dtype.type(ret / rcount)
WARNING:absl:At this time, the v2.11+ optimizer `tf.keras.optimizers.Adam` runs slowly on M1/M2 Macs, please use the legacy Keras optimizer instead, located at `tf.keras.optimizers.legacy.Adam`.
161/203 [======================>.......] - ETA: 0s - loss: 3.7583e-05e-05 - val_loss: 1.9904e-056
 49/203 [======>.......................] - ETA: 1s - loss: 2.2150e-05e-05 - val_loss: 1.7167e-066
203/203 [==============================] - 1s 7ms/step - loss: 2.3210e-05 - val_loss: 1.0637e-056
203/203 [==============================] - 1s 7ms/step - loss: 1.9002e-05 - val_loss: 2.3175e-046
137/203 [===================>..........] - ETA: 0s - loss: 1.6877e-05e-05 - val_loss: 2.3175e-046
Epoch 1/50=============================] - 1s 7ms/step - loss: 1.7796e-05 - val_loss: 5.9634e-066
Epoch 1/50=============================] - 1s 7ms/step - loss: 1.7796e-05 - val_loss: 5.9634e-066
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/fromnumeric.py:3464: RuntimeWarning: Mean of empty slice.
  return _methods._mean(a, axis=axis, dtype=dtype,
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:192: RuntimeWarning: invalid value encountered in divide
  ret = ret.dtype.type(ret / rcount)
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:269: RuntimeWarning: Degrees of freedom <= 0 for slice
  ret = _var(a, axis=axis, dtype=dtype, out=out, ddof=ddof,
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:226: RuntimeWarning: invalid value encountered in divide
  arrmean = um.true_divide(arrmean, div, out=arrmean,
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:261: RuntimeWarning: invalid value encountered in divide
  ret = ret.dtype.type(ret / rcount)
WARNING:absl:At this time, the v2.11+ optimizer `tf.keras.optimizers.Adam` runs slowly on M1/M2 Macs, please use the legacy Keras optimizer instead, located at `tf.keras.optimizers.legacy.Adam`.
Epoch 1/50=============================] - 1s 7ms/step - loss: 1.7796e-05 - val_loss: 5.9634e-066
Epoch 3/50=============================] - 1s 7ms/step - loss: 1.7796e-05 - val_loss: 5.9634e-066
Epoch 3/50=============================] - 1s 7ms/step - loss: 1.7796e-05 - val_loss: 5.9634e-066
Epoch 5/50=============================] - 1s 7ms/step - loss: 1.7796e-05 - val_loss: 5.9634e-066
Epoch 7/50=============================] - 1s 7ms/step - loss: 1.7796e-05 - val_loss: 5.9634e-066
Epoch 9/50=============================] - 1s 7ms/step - loss: 1.7796e-05 - val_loss: 5.9634e-066
Epoch 9/50=============================] - 1s 7ms/step - loss: 1.7796e-05 - val_loss: 5.9634e-066
Epoch 11/50============================] - 1s 7ms/step - loss: 1.7796e-05 - val_loss: 5.9634e-066
Epoch 11/50============================] - 1s 7ms/step - loss: 1.7796e-05 - val_loss: 5.9634e-066
Epoch 13/50============================] - 1s 7ms/step - loss: 1.7796e-05 - val_loss: 5.9634e-066
Training completed with early stopping to prevent overfitting. 1.7796e-05 - val_loss: 5.9634e-066
Training completed with early stopping to prevent overfitting. 1.7796e-05 - val_loss: 5.9634e-066
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/fromnumeric.py:3464: RuntimeWarning: Mean of empty slice.
  return _methods._mean(a, axis=axis, dtype=dtype,
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:192: RuntimeWarning: invalid value encountered in divide
  ret = ret.dtype.type(ret / rcount)
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:269: RuntimeWarning: Degrees of freedom <= 0 for slice
  ret = _var(a, axis=axis, dtype=dtype, out=out, ddof=ddof,
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:226: RuntimeWarning: invalid value encountered in divide
  arrmean = um.true_divide(arrmean, div, out=arrmean,
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:261: RuntimeWarning: invalid value encountered in divide
  ret = ret.dtype.type(ret / rcount)
WARNING:absl:At this time, the v2.11+ optimizer `tf.keras.optimizers.Adam` runs slowly on M1/M2 Macs, please use the legacy Keras optimizer instead, located at `tf.keras.optimizers.legacy.Adam`.
 67/203 [========>.....................] - ETA: 1s - loss: 4.8774e-04e-05 - val_loss: 5.9634e-066
198/203 [============================>.] - ETA: 0s - loss: 1.9835e-04e-05 - val_loss: 5.9634e-066
Epoch 3/50===========================>.] - ETA: 0s - loss: 1.9835e-04e-05 - val_loss: 5.9634e-066
Epoch 3/50===========================>.] - ETA: 0s - loss: 1.9835e-04e-05 - val_loss: 5.9634e-066
Epoch 3/50===========================>.] - ETA: 0s - loss: 1.9835e-04e-05 - val_loss: 5.9634e-066
Epoch 3/50===========================>.] - ETA: 0s - loss: 1.9835e-04e-05 - val_loss: 5.9634e-066
Epoch 3/50===========================>.] - ETA: 0s - loss: 1.9835e-04e-05 - val_loss: 5.9634e-066
Epoch 3/50===========================>.] - ETA: 0s - loss: 1.9835e-04e-05 - val_loss: 5.9634e-066
Epoch 3/50===========================>.] - ETA: 0s - loss: 1.9835e-04e-05 - val_loss: 5.9634e-066
Epoch 3/50===========================>.] - ETA: 0s - loss: 1.9835e-04e-05 - val_loss: 5.9634e-066
Epoch 3/50===========================>.] - ETA: 0s - loss: 1.9835e-04e-05 - val_loss: 5.9634e-066
Epoch 12/50==========================>.] - ETA: 0s - loss: 1.9835e-04e-05 - val_loss: 5.9634e-066
All data for ZBRA saved successfully to Excel.: 0s - loss: 1.9835e-04e-05 - val_loss: 5.9634e-066
All data for ZBRA saved successfully to Excel.: 0s - loss: 1.9835e-04e-05 - val_loss: 5.9634e-066
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/fromnumeric.py:3464: RuntimeWarning: Mean of empty slice.
  return _methods._mean(a, axis=axis, dtype=dtype,
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:192: RuntimeWarning: invalid value encountered in divide
  ret = ret.dtype.type(ret / rcount)
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:269: RuntimeWarning: Degrees of freedom <= 0 for slice
  ret = _var(a, axis=axis, dtype=dtype, out=out, ddof=ddof,
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:226: RuntimeWarning: invalid value encountered in divide
  arrmean = um.true_divide(arrmean, div, out=arrmean,
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:261: RuntimeWarning: invalid value encountered in divide
  ret = ret.dtype.type(ret / rcount)
All data for ZBRA saved successfully to Excel.: 0s - loss: 1.9835e-04e-05 - val_loss: 5.9634e-066
WARNING:absl:At this time, the v2.11+ optimizer `tf.keras.optimizers.Adam` runs slowly on M1/M2 Macs, please use the legacy Keras optimizer instead, located at `tf.keras.optimizers.legacy.Adam`.
All data for ZBRA saved successfully to Excel.: 0s - loss: 1.9835e-04e-05 - val_loss: 5.9634e-066
All data for ZBRA saved successfully to Excel.: 0s - loss: 1.9835e-04e-05 - val_loss: 5.9634e-066
All data for ZBRA saved successfully to Excel.: 0s - loss: 1.9835e-04e-05 - val_loss: 5.9634e-066
All data for ZBRA saved successfully to Excel.: 0s - loss: 1.9835e-04e-05 - val_loss: 5.9634e-066
Epoch 6/50or ZBRA saved successfully to Excel.: 0s - loss: 1.9835e-04e-05 - val_loss: 5.9634e-066
Epoch 6/50or ZBRA saved successfully to Excel.: 0s - loss: 1.9835e-04e-05 - val_loss: 5.9634e-066
Epoch 9/50or ZBRA saved successfully to Excel.: 0s - loss: 1.9835e-04e-05 - val_loss: 5.9634e-066
All data for ZBRA saved successfully to Excel.: 0s - loss: 1.9835e-04e-05 - val_loss: 5.9634e-066
All data for ZBRA saved successfully to Excel.: 0s - loss: 1.9835e-04e-05 - val_loss: 5.9634e-066
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/fromnumeric.py:3464: RuntimeWarning: Mean of empty slice.
  return _methods._mean(a, axis=axis, dtype=dtype,
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:192: RuntimeWarning: invalid value encountered in divide
  ret = ret.dtype.type(ret / rcount)
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:269: RuntimeWarning: Degrees of freedom <= 0 for slice
  ret = _var(a, axis=axis, dtype=dtype, out=out, ddof=ddof,
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:226: RuntimeWarning: invalid value encountered in divide
  arrmean = um.true_divide(arrmean, div, out=arrmean,
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:261: RuntimeWarning: invalid value encountered in divide
  ret = ret.dtype.type(ret / rcount)
All data for ZBRA saved successfully to Excel.: 0s - loss: 1.9835e-04e-05 - val_loss: 5.9634e-066
WARNING:absl:At this time, the v2.11+ optimizer `tf.keras.optimizers.Adam` runs slowly on M1/M2 Macs, please use the legacy Keras optimizer instead, located at `tf.keras.optimizers.legacy.Adam`.
All data for ZBRA saved successfully to Excel.: 0s - loss: 1.9835e-04e-05 - val_loss: 5.9634e-066
Epoch 5/50or ZBRA saved successfully to Excel.: 0s - loss: 1.9835e-04e-05 - val_loss: 5.9634e-066
Epoch 5/50or ZBRA saved successfully to Excel.: 0s - loss: 1.9835e-04e-05 - val_loss: 5.9634e-066
Training completed with early stopping to prevent overfitting.835e-04e-05 - val_loss: 5.9634e-066
Training completed with early stopping to prevent overfitting.835e-04e-05 - val_loss: 5.9634e-066
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/fromnumeric.py:3464: RuntimeWarning: Mean of empty slice.
  return _methods._mean(a, axis=axis, dtype=dtype,
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:192: RuntimeWarning: invalid value encountered in divide
  ret = ret.dtype.type(ret / rcount)
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:269: RuntimeWarning: Degrees of freedom <= 0 for slice
  ret = _var(a, axis=axis, dtype=dtype, out=out, ddof=ddof,
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:226: RuntimeWarning: invalid value encountered in divide
  arrmean = um.true_divide(arrmean, div, out=arrmean,
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:261: RuntimeWarning: invalid value encountered in divide
  ret = ret.dtype.type(ret / rcount)
WARNING:absl:At this time, the v2.11+ optimizer `tf.keras.optimizers.Adam` runs slowly on M1/M2 Macs, please use the legacy Keras optimizer instead, located at `tf.keras.optimizers.legacy.Adam`.
Training completed with early stopping to prevent overfitting.835e-04e-05 - val_loss: 5.9634e-066
Training completed with early stopping to prevent overfitting.835e-04e-05 - val_loss: 5.9634e-066
Training completed with early stopping to prevent overfitting.835e-04e-05 - val_loss: 5.9634e-066
Epoch 5/50ompleted with early stopping to prevent overfitting.835e-04e-05 - val_loss: 5.9634e-066
Epoch 5/50ompleted with early stopping to prevent overfitting.835e-04e-05 - val_loss: 5.9634e-066
Epoch 5/50ompleted with early stopping to prevent overfitting.835e-04e-05 - val_loss: 5.9634e-066
All data for ZBRA saved successfully to Excel.ent overfitting.835e-04e-05 - val_loss: 5.9634e-066
All data for ZBRA saved successfully to Excel.ent overfitting.835e-04e-05 - val_loss: 5.9634e-066
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/fromnumeric.py:3464: RuntimeWarning: Mean of empty slice.
  return _methods._mean(a, axis=axis, dtype=dtype,
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:192: RuntimeWarning: invalid value encountered in divide
  ret = ret.dtype.type(ret / rcount)
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:269: RuntimeWarning: Degrees of freedom <= 0 for slice
  ret = _var(a, axis=axis, dtype=dtype, out=out, ddof=ddof,
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:226: RuntimeWarning: invalid value encountered in divide
  arrmean = um.true_divide(arrmean, div, out=arrmean,
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:261: RuntimeWarning: invalid value encountered in divide
  ret = ret.dtype.type(ret / rcount)
WARNING:absl:At this time, the v2.11+ optimizer `tf.keras.optimizers.Adam` runs slowly on M1/M2 Macs, please use the legacy Keras optimizer instead, located at `tf.keras.optimizers.legacy.Adam`.
All data for ZBRA saved successfully to Excel.ent overfitting.835e-04e-05 - val_loss: 5.9634e-066
All data for ZBRA saved successfully to Excel.ent overfitting.835e-04e-05 - val_loss: 5.9634e-066
All data for ZBRA saved successfully to Excel.ent overfitting.835e-04e-05 - val_loss: 5.9634e-066
Epoch 6/50or ZBRA saved successfully to Excel.ent overfitting.835e-04e-05 - val_loss: 5.9634e-066
Epoch 6/50or ZBRA saved successfully to Excel.ent overfitting.835e-04e-05 - val_loss: 5.9634e-066
Epoch 6/50or ZBRA saved successfully to Excel.ent overfitting.835e-04e-05 - val_loss: 5.9634e-066
Epoch 10/50r ZBRA saved successfully to Excel.ent overfitting.835e-04e-05 - val_loss: 5.9634e-066
Epoch 10/50r ZBRA saved successfully to Excel.ent overfitting.835e-04e-05 - val_loss: 5.9634e-066
Epoch 10/50r ZBRA saved successfully to Excel.ent overfitting.835e-04e-05 - val_loss: 5.9634e-066
All data for ZBRA saved successfully to Excel.ent overfitting.835e-04e-05 - val_loss: 5.9634e-066
All data for ZBRA saved successfully to Excel.ent overfitting.835e-04e-05 - val_loss: 5.9634e-066
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/fromnumeric.py:3464: RuntimeWarning: Mean of empty slice.
  return _methods._mean(a, axis=axis, dtype=dtype,
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:192: RuntimeWarning: invalid value encountered in divide
  ret = ret.dtype.type(ret / rcount)
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:269: RuntimeWarning: Degrees of freedom <= 0 for slice
  ret = _var(a, axis=axis, dtype=dtype, out=out, ddof=ddof,
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:226: RuntimeWarning: invalid value encountered in divide
  arrmean = um.true_divide(arrmean, div, out=arrmean,
/Users/ruthikkale/anaconda3/lib/python3.11/site-packages/numpy/core/_methods.py:261: RuntimeWarning: invalid value encountered in divide
  ret = ret.dtype.type(ret / rcount)
WARNING:absl:At this time, the v2.11+ optimizer `tf.keras.optimizers.Adam` runs slowly on M1/M2 Macs, please use the legacy Keras optimizer instead, located at `tf.keras.optimizers.legacy.Adam`.
All data for ZBRA saved successfully to Excel.ent overfitting.835e-04e-05 - val_loss: 5.9634e-066
All data for ZBRA saved successfully to Excel.ent overfitting.835e-04e-05 - val_loss: 5.9634e-066
All data for ZBRA saved successfully to Excel.ent overfitting.835e-04e-05 - val_loss: 5.9634e-066